### ISSUE TITLE
what tricks can be applied in yolo?

### ISSUE THREAD

**Issue by zimurui** [qqwweee/keras-yolo3#302]
Hi, I copy and did some modification on  your code to train in  VOC2007. After training a days,  the train loss seems drop diffuculy,  the train loss is about fixed on 30, the val loss is 60. I used three GTX 1080 Ti, set batch size 32, leraing rate is 1e-5. And I don't know how to use tricks to YOLOv3..
can you tell me how long do you train this code? And what level can the loss can converges on VOC and COCO dataset?

**Comment by Stinky-Tofu** [comment]
@zimurui You can reference my code https://github.com/Stinky-Tofu/Stronger-yolo , there are lots of SOAT tricks, and have achieved 90mAP on VOC2007
